pkg/eosclient/eosgrpc/versions.go

```diff
@@ -59,7 +59,7 @@ func (c *Client) RollbackToVersion(ctx context.Context, auth eosclient.Authoriza
 	if resp.GetError() != nil {
 		log.Info().Str("func", "RollbackToVersion").Int64("errcode", resp.GetError().Code).Str("errmsg", resp.GetError().Msg).Msg("grpc response")
 	}
-	return err
+	return nil
```

Comment:
What is the implication of this?

Reply:
nothing, because we already return if we have an error earlier :)
now it also always returned nil, just implicitly